### PR TITLE
Fix templating errors in jaas-login module - issue #5731

### PR DESCRIPTION
**Pull request by cammoraton**
Fix removal of whitespace causing unintended scope changes in the resulting rendered file.

Fix inconsistency in quoting when driving from environmental variables.

### CODE REVIEW

File: docker/official/remco/templates/jaas-loginmodule.conf [hunk 1]
```diff
@@ -22,36 +22,36 @@
...
+        userLastNameAttribute="{{ getv("/rundeck/jaas/ldap/userlastnameattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/userfirstnameattribute") %}
+        userFirstNameAttribute="{{ getv("/rundeck/jaas/ldap/userfirstnameattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/useremailattribute") %}
+        userEmailAttribute="{{ getv("/rundeck/jaas/ldap/useremailattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/nestedgroups") -%}
+        nestedGroups="{{ getv("/rundeck/jaas/ldap/nestedGroups") }}"
```
Comment (gschueler) [inline docker/official/remco/templates/jaas-loginmodule.conf]: was this case change intentional?
```suggestion
        nestedGroups="{{ getv("/rundeck/jaas/ldap/nestedgroups") }}"
```

Comment (cammoraton) [reply docker/official/remco/templates/jaas-loginmodule.conf]: Yes. remco was failing to render when RUNDECK_JAAS_LDAP_NESTEDGROUP was set with the mixed case in getv.

File: docker/official/remco/templates/jaas-loginmodule.conf [hunk 2]
```diff
@@ -22,36 +22,36 @@
         rolePrefix="{{ getv("/rundeck/jaas/ldap/roleprefix", "") }}"
         cacheDurationMillis="{{ getv("/rundeck/jaas/ldap/cachedurationmillis", "600000") }}"
         reportStatistics="{{ getv("/rundeck/jaas/ldap/reportstatistics", "true") }}"
-    {% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") -%}
+{% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") %}
```
Comment (ProTip) [inline docker/official/remco/templates/jaas-loginmodule.conf]: Testing this it appears the spacing in the if/else statements does not impact the rendered output(tested with remco master and the pinned version). Are you seeing something different?

File: docker/official/remco/templates/jaas-loginmodule.conf [hunk 3]
```diff
@@ -22,36 +22,36 @@
...
+{% if exists("/rundeck/jaas/ldap/userlastnameattribute") %}
+        userLastNameAttribute="{{ getv("/rundeck/jaas/ldap/userlastnameattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/userfirstnameattribute") %}
+        userFirstNameAttribute="{{ getv("/rundeck/jaas/ldap/userfirstnameattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/useremailattribute") %}
+        userEmailAttribute="{{ getv("/rundeck/jaas/ldap/useremailattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/nestedgroups") -%}
```
Comment (ProTip) [inline docker/official/remco/templates/jaas-loginmodule.conf]:
```suggestion
{% if exists("/rundeck/jaas/ldap/nestedgroups") %}
```

File: docker/official/remco/templates/jaas-loginmodule.conf (hunk 2 above)
Comment (cammoraton) [reply docker/official/remco/templates/jaas-loginmodule.conf]: It's fine individually, but when I apply multiple settings the whitespace stripping at the end of the if can cause them to end up clumped up at the start of the line.

So setting:
```
RUNDECK_JAAS_LDAP_ROLEMEMBERATTRIBUTE=member
RUNDECK_JAAS_LDAP_ROLEOBJECTCLASS=groupOfNames
RUNDECK_JAAS_LDAP_SUPPLEMENTALROLES=user
```

Results in:
```
        cacheDurationMillis="600000"
        reportStatistics="true"
userLastNameAttribute=sn
userFirstNameAttribute=givenName
userEmailAttribute=mail
```
being rendered in the template. This doesn't break rundeck starting but does break those attributes.

**Pull request merged**